resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1172,6 +1181,31 @@ private[spark] class Client(
     }.mkString("")
   }
 
+  private def getDriverLogsLink(appId: String): Option[(String, String)] = {
+    val baseRmUrl = WebAppUtils.getRMWebAppURLWithScheme(hadoopConf)
+    val response = ClientBuilder.newClient()
+      .target(baseRmUrl)
+      .path("ws").path("v1").path("cluster").path("apps").path(appId).path("appattempts")
```

Comment:
Looks like there is no way to reference these from yarn project directly ?
It does not look like `RMWSConsts` is a public api ?

Reply:
Yeah, I do not think so. Even in Hadoop itself (`LogsCLI`), this same approach is taken: https://github.com/apache/hadoop/blob/master/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/cli/LogsCLI.java#L368